usaon_vta_survey/templates/macros/javascript_delete.j2

```diff
@@ -1,6 +1,6 @@
 {% macro add_javascript__delete() -%}
   <script>
-    function remove_relationship(url, response) {
+    function remove(url, response) {
```

Comment:
I feel the JS function should match the macro name, e.g. call it `delete`, or change the macro to `add_javascript__remove`

Reply:
good call, it was doing a weird highlight when I tried calling the function `delete` so I figured it may have been some sort of keyword and didnt wanna mess with that.